The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/6752353988/job/18357629400) [(plain)](https://github.com/rust-lang/rust/commit/e45701fce5c60e08e85c47fc33265f8a6c3a6946/checks/18357629400/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=0xalpharush
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_d5a3a763-f014-4cf3-b65a-e023acde94cd
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=18a08b7c549663eabaadb3757468910f577832ed
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_d5a3a763-f014-4cf3-b65a-e023acde94cd
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_d5a3a763-f014-4cf3-b65a-e023acde94cd
GITHUB_TRIGGERING_ACTOR=0xalpharush
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/117563/merge
GITHUB_WORKFLOW_SHA=18a08b7c549663eabaadb3757468910f577832ed
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---

---- library/alloc/src/boxed.rs - boxed::Box<T,A>::into_raw (line 1048) stdout ----
error[E0425]: cannot find value `ptr` in this scope
##[error] --> library/alloc/src/boxed.rs:1051:20
error: doctest failed, to rerun pass `-p alloc --doc`
  |
6 | drop(Box::from_raw(ptr));
  |                    ^^^ not found in this scope

```

</details>

--- a/library/alloc/src/boxed.rs
+++ b/library/alloc/src/boxed.rs
@@ -1038,10 +1038,18 @@ impl<T: ?Sized, A: Allocator> Box<T, A> {
     /// use std::ptr;
     ///
     /// let x = Box::new(String::from("Hello"));
-    /// let p = Box::into_raw(x);
+    /// let ptr = Box::into_raw(x);
     /// unsafe {
-    ///     ptr::drop_in_place(p);
-    ///     dealloc(p as *mut u8, Layout::new::<String>());
+    ///     ptr::drop_in_place(ptr);
+    ///     dealloc(ptr as *mut u8, Layout::new::<String>());
+    /// }
+    /// ```
+    /// Note: This is equivalent to the following:
+    /// ```
+    /// let x = Box::new(String::from("Hello"));
+    /// let ptr = Box::into_raw(x);
+    /// unsafe {
+    ///     drop(Box::from_raw(ptr));
     /// }
     /// ```
     ///

